files/es/web/html/element/iframe/index.md

```diff
@@ -0,0 +1,274 @@
+
+### Atributos no estándar
+
+- {{htmlattrdef("mozbrowser")}} {{non-standard_inline}}
+
+  - :&#x20;
+
+    > **Nota:** See {{bug(1318532)}} for exposing this to WebExtensions in Firefox.
+
+    Makes the `<iframe>` act like a top-level browser window. See [Browser API](/es/docs/Mozilla/Gecko/Chrome/API/Browser_API) for details.
+    **Available only to [WebExtensions](/es/docs/Mozilla/Add-ons/WebExtensions).**
+
+- {{ htmlattrdef("mozallowfullscreen") }} {{ non-standard_inline() }}
+  - : In Gecko 9.0 or later, this attribute can be set to `true` if the frame is allowed to be placed into full screen mode by calling its {{ domxref("element.mozRequestFullScreen()") }} method. If this isn't set, the element can't be placed into full screen mode.
+- {{ htmlattrdef("webkitallowfullscreen") }} {{ non-standard_inline() }}
+  - : In Chrome 17 or later (and maybe earlier), this attribute can be set to `true` if the frame is allowed to be placed into full screen mode by calling its {{ domxref("element.webkitRequestFullScreen()") }} method. If this isn't set, the element can't be placed into full screen mode.
+- {{ htmlattrdef("mozapp") }} {{ non-standard_inline() }}
+  - : For frames hosting an [open web app](/en/Apps), this specifies the URL of the [app manifest](/en/Apps/Manifest). This ensures that the app is loaded with the right permissions. See [Using the Browser API](/en/DOM/Using_the_Browser_API) for details. Available in Gecko 13.0 and later.
+- {{ htmlattrdef("remote") }} {{ non-standard_inline() }}
+  - : Load the frame's page in a separate content process.
+
+## Secuencia de comandos
+
+Inline frames, like {{HTMLElement("frame")}} elements, are included in the {{domxref("window.frames")}} pseudo-array.
+
+With the DOM {{domxref("HTMLIFrameElement")}} object, scripts can access the {{domxref("window")}} object of the framed resource via the {{domxref("HTMLIFrameElement.contentWindow", "contentWindow")}} property. The {{domxref("HTMLIFrameElement.contentDocument", "contentDocument")}} property refers to the `document` inside the `<iframe>`, same as `contentWindow.document`.
+
+From the inside of a frame, a script can get a reference to its parent window with {{domxref("window.parent")}}.
+
+Script access to a frame's content is subject to the {{Glossary("same-origin policy")}}. Scripts cannot access most properties in other `window` objects if the script was loaded from a different origin, including scripts inside a frame accessing the frame's parent. Cross-origin communication can be achieved using {{domxref("Window.postMessage()")}}.
+
+## Ejemplos
+
+### Un \<iframe> simple
+
+Un `<iframe>` en acción. Después de crear el marco, cuando el usuario hace clic en un botón, su título se muestra en una alerta.
+
+#### HTML
+
+```
+<iframe src="https://mdn-samples.mozilla.org/snippets/html/iframe-simple-contents.html" title="iframe Example 1" width="400" height="300">
+  <p>Your browser does not support iframes.</p>
+</iframe>
+```
```

Comment:
````suggestion
```html
<iframe src="https://mdn-samples.mozilla.org/snippets/html/iframe-simple-contents.html" title="iframe Example 1" width="400" height="300">
  <p>Your browser does not support iframes.</p>
</iframe>
```
````